packages/cache/package-lock.json

```diff
@@ -1,625 +1,13 @@
 {
   "name": "@actions/cache",
-  "version": "3.0.4",
-  "lockfileVersion": 2,
```

Comment:
Why did we change lockfile version?

Reply:
This was changed on using `npm version patch` command. Reverting.